backend/zero_alloc_info.ml

```diff
@@ -0,0 +1,52 @@
+module String = Misc.Stdlib.String
```

Comment:
(For history / `git blame`, it would probably
be better to have this as a rename of
`checks.ml` rather than a deletion/addition.)

Reply:
`git show` on the first commit does show it as a move.

Reply:
That might be a GitHub UI issue then.